fecha.js

```diff
@@ -237,9 +238,21 @@
 
     mask = fecha.masks[mask] || mask || fecha.masks['default'];
 
-    return mask.replace(token, function ($0) {
+    var literals = [];
+
+    //Make literals inactive by replacing them with _
+    mask = mask.replace(literal, function($0, $1) {
+      literals.push($1);
+      return '_';
+    });
+    //Apply formatting rules
```

Comment:
Thanks, I tried to follow the code conventions in the library, but this somehow escaped my attention, will add an extra space.